src/async-components/views/dialogs/keybackup/CreateKeyBackupDialog.js

```diff
@@ -265,8 +264,15 @@ export default React.createClass({
         }
 
         return <div>
-            <p>{_t("Secure your encrypted message history with a Recovery Passphrase.")}</p>
-            <p>{_t("You'll need it if you log out or lose access to this device.")}</p>
+            <p>{_t(
+                "<b>Warning</b>: you should only set up key backup from a trusted computer.", {},
+                { b: sub => <b>{sub}</b> },
+            )}</p>
+            <p>{_t(
+                "We'll store an encrypted copy of your keys on our server. " +
+                "Protect your backup with a passphrase to keep it secure.",
+            )}</p>
+            <p>{_t("For maximum security, this should be different from your account password.")}</p>
```

Comment:
recommendation: "This should be unique and not used anywhere else".

(I think @lampholder is responsible for the wording though?)

Reply:
I like the suggestion, but am afraid its broader truth is missing the specific critical point (and at the risk of overthinking it, people might be confused by 'not used anywhere else' - _is_ somewhere else in Riot 'somewhere else'?).

I can't immediately think how to adopt the spirit of the suggestion without losing that specific key point (or making it too wordy) so I'm inclined to stick with the original at the moment :S